airbyte-integrations/connectors/source-intercom/source_intercom/source.py

```diff
@@ -193,6 +194,9 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
             return super().next_page_token(response)
         return None
 
+    def need_use_standard(self):
+        return not self.can_use_scroll() or self._use_standard
+
     def can_use_scroll(self):
         """Check backoff count"""
         return self._backoff_count <= 3
```

Comment:
can we use one of their functions? and remove another.  it seems the function `can_use_scroll` is used into the single place only and we can merge it with "need_use_standard"

Reply:
I think better to divide these two functions. `can_use_scroll` is also used in pagination when we use scroll.

Reply:
Ok